scss/layout/sidebar.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../stanford_basic/scss/components/sidebar/sidebar";
+@import "config/config";
+//// Decanter config.
+@import 'decanter/core/utilities/decanter-utilities';
+// Grab our mixins.
+@import "utilities/utilities";
+
+body.with-banner {
+  header {
+    //min-height: 14px;
+
+    #block-matson-branding {
+      position: absolute;
+      padding: 20px;
+      left: 128px;
```

Comment:
This hard fix to the left edge doesn't really mesh with the responsive layouts and may prove to be a problem when trying to figure out the paragraph offsets. It would be ideal of this element would be in a responsive container and left align as if it were in the `basic` layout for example.